lib/state_of_elixir_web/controllers/home/start.html.heex

```diff
@@ -1,36 +1,35 @@
-<div class="min-h-screen flex flex-col items-center mt-10">
-  <div class="text-center text-5xl md:text-8xl">
-    <div class="header">
-      <p class="mt-8">State of Elixir</p>
-      <p class="mt-8">2023</p>
-    </div>
-  </div>
+<div class="flex flex-col justify-between items-start flex-shrink-0 gap-6 sm:gap-8 mt-2 sm:mt-10 xs:max-w-xs sm:max-w-none">
```

Comment:
We went desktop-first, which made this ugly, but I expect most traffic to be desktop, so we should be fine w/ this.